include/fastdds/dds/topic/TopicDataType.hpp

```diff
@@ -130,7 +106,7 @@ class TopicDataType
      */
     FASTDDS_EXPORTED_API virtual std::function<uint32_t()> getSerializedSizeProvider(
```

Comment:
Can we check whether this could return `uint32_t` directly, instead of a function?

Reply:
Done